backend/app/assets/stylesheets/spree/backend/shared/_typography.scss

```diff
@@ -1,5 +1,10 @@
 // Base
 //--------------------------------------------------------------
+
+html {
+  font-size: $font-size-root;
```

Comment:
As long as we set `$font-size-base` to `$body-font-size` everything should be fine.